### DISCUSSION TITLE
连接问题

### DISCUSSION THREAD

**Discussion by Evan-tc** [Yue-Yang/ChatGPT-Siri#81] [General]
用了魔法，还 是提示发生了ssl错误 无法建立与该服务器的安全连接，但是网页能进chatgpt官网

**Comment by Yue-Yang** [comment]
看看你的帐户是不是被ban了，看下邮件。然后看下API用量是否有余额：https://platform.openai.com/account/usage

**Comment by pipiqiqi** [comment]
我也遇到一样的问题，有解吗？网页能正常使用GPT

**Comment by CarlCit** [comment]
我的是 Siri 使用连接不上，捷径打字用没问题，账号正常。

**Comment by ZyoungInc** [comment]
我也困惑了很久，debug了clash和网络跳转终于找到原因了。当你使用梯子时，请不要在你的网络环境启用ipv6，问题就解决了。ipv6+SSL的代理目前还有许多问题

**Reply by cccClyde** [reply]
服了都，也困扰了我两三天，还好搜到这里了，亲测有效，感谢。
